pkg/cmd/cli/describe/projectstatus.go

```diff
@@ -233,6 +233,9 @@ func (d *ProjectStatusDescriber) Describe(namespace, name string) (string, error
 			printLines(out, indent, 0, describeMonopod(f, monopod.Pod)...)
 		}
 
+		bcPipelines, _ := graphview.AllImagePipelinesFromBuildConfig(g, graphview.IntSet{})
```

Comment:
> What do we do when there are multiple inputs to the same build (because of image sources)?

Hmmm, that wasn't a thing when I was dealing with this last.